src/scene/mesh-instance.js

```diff
@@ -58,12 +58,26 @@ class InstancingData {
     /** @type {VertexBuffer|null} */
     vertexBuffer = null;
 
+    /**
+     * True if the vertex buffer is destroyed when the mesh instance is destroyed.
+     *
+     * @type {boolean}
```

Comment:
Should this be `@private`?

Reply:
Ah, you reference it in `glb-container-resource.js`. So maybe `@ignore` instead?

Reply:
we removed all `@ignore` tags from non-exported classes.

Reply:
Oh, this is `InstancingData`. I thought it was `MeshInstance`. OK, forget it.